packages/ember-headless-form/src/-private/components/control/checkbox.gts

```diff
@@ -43,4 +44,16 @@ export default class HeadlessFormControlCheckboxComponent extends Component<Head
   handleInput(e: Event | InputEvent): void {
     this.args.setValue((e.target as HTMLInputElement).checked);
   }
+
+  <template>
+    <input
+      name={{@name}}
+      type="checkbox"
```

Comment:
I'm surprised to see double quotes here, as we're now (sort of) in JS, and they were single quotes in the hbs... 🤷

Reply:
Oh, didn't realise this before. Probably our prettier config is a bit inconsistent. I did wonder why we have single quotes in hbs before (which in other projects were usually double quotes)...